lib/crypto_importers/mappers/coinmetro_mapper.rb

```diff
@@ -0,0 +1,26 @@
+class CoinmetroMapper < BaseMapper
+  tag Tag::COIN_METRO
+  mappings [
+    {
+      id: 'coinmetro-transactions',
+      required_headers: %w[Currency Date Description Amount Fees Price Pair Other\ Currency Other\ Amount],
+      optional_headers: %w[IBAN Transaction\ Hash Additional\ Info],
+      header_mappings: {
+        date: 'Date',
+        amount: 'Amount',
+        currency: 'Currency',
+        txhash: 'Transaction hash',
+        description: 'Description',
+        fee_currency: 'Currency',
+        fee_amount: 'Fees'
+      },
+      group: {
+        by_hash: ->(_mapped_row, raw_row) { raw_row['Date'] },
+        eligible: ->(_mapped_row, raw_row) { raw_row['Description'].match?(/order/i) },
```

Comment:
We should group by hash:
```suggestion
        by_hash: ->(_mapped, row) { row['Description'] },
```

Reply:
@Vlad-Frolov updated